### PR TITLE
[Fix] localStorage and map scripts - MS Edge compatibility

### PR DESCRIPTION
**Pull request by TomProkop**
## Motivation and Context

Map was not loading properly in MS Edge.

## How Has This Been Tested?
I ran the app in Chrome and Edge

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by MonkeyLeeT** [comment]
#1694

**Comment by MonkeyLeeT** [comment]
#1694 #1725 @TomProkop  @Chlodochar  Please take a look and have an agreement.

File: static/map.js [hunk 1]
```diff
@@ -465,10 +465,10 @@ function clearOutOfBoundsMarkers(markers) {
...
 function updateMap() {
 
-    var loadPokemon = localStorage.showPokemon || true;
-    var loadGyms = localStorage.showGyms || true;
-    var loadPokestops =  localStorage.showPokestops || false;
-    var loadScanned = localStorage.showScanned || false;
+    var loadPokemon = localStorage['showPokemon'] || true;
+    var loadGyms = localStorage['showGyms'] || true;
+    var loadPokestops =  localStorage['showPokestops'] || false;
+    var loadScanned = localStorage['showScanned'] || false;
```
Comment (Chlodochar) [inline static/map.js]: This doesn't fix the problems with defaults of the `show*` options.

**Comment by Chlodochar** [comment]
This PR fixes problems with Edge, but doesn't fix the problems with incorrect logic being used on the localStorage (it still ignores defaults and doesn't properly parse the raw localStorage values). #1694 does and should also work for Edge (it includes the same fix), but I have not tested this (nor am I able to).

File: templates/map.html [hunk 2]
```diff
@@ -133,7 +133,7 @@
 		var center_lng = {{lng}};
 	</script>
 
-	<script async defer src="https://maps.googleapis.com/maps/api/js?key={{ gmaps_key }}&callback=initMap&libraries=places"></script>
 	<script type="text/javascript" src="static/map.js"></script>
+	<script async defer src="https://maps.googleapis.com/maps/api/js?key={{ gmaps_key }}&callback=initMap&libraries=places"></script>
```
Comment (Chlodochar) [inline templates/map.html]: Why load gmaps after maps.js? Wouldn't that cause potential problems?

**Pull request closed without merging**